configure.sh

```diff
@@ -34,7 +34,21 @@ function write_action_env_to_bazelrc() {
   write_to_bazelrc "build --action_env $1=\"$2\""
 }
 
+function ask_before_install() {
+	req_name=`cat requirements.txt`
+	while true; do
+		echo "Installing $req_name.."
+	    read -p "Proceed ?(Y/N) " yn
+	    case $yn in
+	        [Yy]* ) echo "Installing.."; break;;
+	        [Nn]* ) echo "Exiting.."; exit;;
+	        * ) echo "Please answer Y or N.";;
+	    esac
+	done
+}
+
 [[ -f .bazelrc ]] && rm .bazelrc
+ask_before_install
```

Comment:
If I say yes to 5 requirements but then say no to 6th, the script will just terminate.

Reply:
I have made a new commit that fixes that issue. Is there anything else that needs to be done?

Reply:
Great work. Can we make it simple? like

```bash
read -r -p "Tensorflow will be upgraded to 2.0. Are You Sure? [Y/n] " reply
case $reply in
    [yY]) echo "Install...";;
    *) echo "Goodbye!"; exit;;
esac
```

Reply:
@facaiy But won't the script break if we change the `requirements.txt` file later on? If that is fine, then I'll make the changes.

Reply:
Yeah, I agree with you. However it looks kind of boring to ask developers to check each item in the requirements file. That would be great if pip supports dry run, or ask users for confirmation when upgrading, https://stackoverflow.com/questions/29531094/how-to-make-pip-dry-run

Because configure.sh is only used by developers, and we might make docker image for addons itself in the future (we don't need to install python library any more), that's why I want to make the script as simple as possible. Anyway, both solutions are fine with me. I'd like to leave the decision to Sean.

@seanpmorgan Sean, could you talk a look? Thank you :-)

Reply:
I agree with Yan that for now we can make it simple to only check for TensorFlow Nightly install instead of every requirement in requirements.txt and then later burn all the requirements in the docker container anyway.

Reply:
Thank you, Armando. @seanpmorgan Hi, Sean, what do you think?

Reply:
Yes, I think the just prompting the question so it's understood is good enough. The more simple we can make config.sh the better

Reply:
@facaiy I have made the prompt simpler as requested.